src/requirements/requirements.ts

```diff
@@ -20,6 +20,7 @@ export const REQUIREMENTS_DATA = [
   {
     icon: Wallet,
     name: "Free",
+    disabled: false,
```

Comment:
Oh, okay hehe, this was out of scope, but I appreciate it. Let's call is `isDisabled` please (just for consistency)